tools/provision

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+if [ "$EUID" -eq 0 ]; then
+    echo "Error: The provision script must not be run as root" >&2
+    exit 1
+fi
+
+#Make the script independent of the location from where it is
+#executed
+PARENT_PATH=$( cd "$(dirname "${BASH_SOURCE}")" ; pwd -P )
+cd "$PARENT_PATH"
+mkdir -p ../var/log
+LOG_PATH="../var/log/zulip_provision.log"
```

Comment:
Okay, can change it. My reasoning behind this was `Vagrantfile`, which originally ran `/srv/zulip/tools/provision.py | sudo tee -a /var/log/zulip_provision.log`.